src/package_class.ts

```diff
@@ -0,0 +1,48 @@
+/* Package class module
+* initialized then passed to the APIs to have data loaded to it and then used by the Runner to calculate score
+*/ 
+export class package_class{ 
```

Comment:
Sorry if this is nitpicky, but could we do a uniform standard for class names in terms of casing and name? As in, we use camelCase for classes (so something like Package) and snake case for variables (like bus_factor, which is what you have).

Reply:
It should be fixed now